eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyFallbackTest.java

```diff
@@ -53,23 +49,42 @@ void putsObjectToLocalCache(@TempDir final Path path) throws Exception {
             new OyCaching(
                 branch,
                 path,
-                name -> {
-                    counter.incrementAndGet();
-                    return new InputOf("[] > main\n");
-                }
+                new OyFake(
```

Comment:
@MikhailLipanin It's better to split the test to 2 cases:
1. check for `get`
2. check for `contains`

Reply:
@volodya-lombrozo, I think this test has the complicated logic even without `contains()` method. I have added TODO to simplify this test.